lms/resources/_js_config/__init__.py

```diff
@@ -667,13 +680,17 @@ def _configure_groups(self, course, assignment):
                 },
             }
 
-    def _get_lti_launch_debug_values(self):
+    def _get_lti_launch_debug_values(
+        self, course: Course, assignment: Assignment | None
+    ):
         """Debug values common to different types of LTI launches."""
         ai = self._application_instance
 
         return {
             "Organization ID": ai.organization.public_id if ai.organization else None,
             "Application Instance ID": ai.id,
+            "Assignment ID": assignment.id if assignment else None,
```

Comment:
Arguably `assignment ID: null` is uglier in the output than not adding that key to the dict. Not sure.

But this is a dictionary thrown in the console, doesn't really matter.

Reply:
I first used that approach, but then I saw that, for the organization ID we do `"Organization ID": ai.organization.public_id if ai.organization else None`, so I followed the same approach.

I don't really have a strong opinion.